src/core/lv_global.h

```diff
@@ -211,6 +211,14 @@ typedef struct _lv_global_t {
     lv_mutex_t lv_general_mutex;
 #endif
 
+#if LV_USE_OS == LV_OS_FREERTOS
+    uint32_t os_idle_time_sum;
```

Comment:
`freertos_idle_time_sum`
It is recommended to use clear naming because only FreeRTOS uses.

Reply:
updates